mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -693,7 +766,7 @@ class ALSSuite
     val data = ratings.toDF
     val model = new ALS().fit(data)
     Seq("nan", "NaN", "Nan", "drop", "DROP", "Drop").foreach { s =>
-      model.setColdStartStrategy(s).transform(data)
+      testTransformer[Rating[Int]](data, model.setColdStartStrategy(s), "prediction") { _ => }
```

Comment:
What's the no-op function at the end for -- just because it requires an argument?